tools/KoreBuild.Console/Commands/DockerFiles/winservercore.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/aspnet:4.6.2
+
+ADD https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe /net461dev.exe
+
+RUN /net461dev.exe /Passive /NoRestart
+
+RUN rm /net461dev.exe 
+
+WORKDIR c:\\repo\\
```

Comment:
You need to remove the trailing backslash, otherwise it seems Docker treats it as a line continuation:

```
Step 5/7 : WORKDIR c:\\repo\ADD . .
```